components/director/internal/domain/api/service.go

```diff
@@ -100,21 +110,28 @@ func (s *service) GetForBundle(ctx context.Context, id string, bundleID string)
 	return apiDefinition, nil
 }
 
-func (s *service) CreateInBundle(ctx context.Context, bundleID string, in model.APIDefinitionInput, spec *model.SpecInput) (string, error) {
+func (s *service) CreateInBundle(ctx context.Context, appId, bundleID string, in model.APIDefinitionInput, spec *model.SpecInput) (string, error) {
+	return s.Create(ctx, appId, &bundleID, nil, in, []*model.SpecInput{spec})
```

Comment:
In which cases can the packageID be empty?

Reply:
When the API is created through the GraphQL api